pkg/okteto/auth.go

```diff
@@ -130,7 +130,7 @@ func queryUser(ctx context.Context, client *graphql.Client, token string) (*q, e
 	var user q
 	q := fmt.Sprintf(`query {
 		user {
-			id,name,email,githubID,token,new,registry,buildkit,certificate
+			id,name,email,externalID,token,new,registry,buildkit,certificate
```

Comment:
this is going to break on all appliances until we release and upgrade everyone. I think we could catch the error and then do a second query with githubID (I think we already do that for other fields)